css/src/components/badge.scss

```diff
@@ -72,7 +74,12 @@ $badge-border-radius: $border-radius-rounded !default;
 	}
 
 	.icon {
-		font-size: $badge-icon-font-size;
+		font-size: $badge-icon-size;
+	}
+
+	&:has(.icon:only-child) {
+		padding: $badge-icon-padding-block;
+		border: none;
```

Comment:
I think `border: none;` should be removed. I think this is the desired stated for these in the high contrast theme:

<img width="1136" alt="image" src="https://github.com/user-attachments/assets/bd7218da-572e-4e11-b40c-001983537769">

Reply:
This also means the the default badges will have a border:

<img width="496" alt="image" src="https://github.com/user-attachments/assets/a9ac0e2a-8769-4a10-ba01-22f0312f7903">

Which if not desired can always be changed to use `badge-clear`.

Reply:
updated